CMakeLists.txt

```diff
@@ -32,6 +32,34 @@ if (CONFIG_LV_ATTRIBUTE_FAST_MEM_USE_IRAM)
     target_compile_definitions(${COMPONENT_LIB} PUBLIC "-DLV_ATTRIBUTE_FAST_MEM=IRAM_ATTR")
 endif()
 
+elseif(ZEPHYR_BASE)
+
+if(CONFIG_LVGL)
+
+set(ZEPHYR_CURRENT_LIBRARY lvgl)
+include_directories(${ZEPHYR_BASE}/lib/gui/lvgl)
```

Comment:
```suggestion
zephyr_include_directories(${ZEPHYR_BASE}/lib/gui/lvgl)
```

Reply:
Actually the two lines are no longer needed, I will remove them